src/edtr-io/plugins/highlight/editor.tsx

```diff
@@ -42,45 +19,23 @@ export function HighlightEditor(props: HighlightProps) {
       setEditThrottled(true)
     }
   }
+
   return throttledEdit || edit ? (
     <>
-      <Textarea
+      <textarea
         value={state.code.value}
         name="text"
         placeholder={i18n.code.placeholder}
         onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => {
           state.code.set(e.target.value)
         }}
+        // make sure editor does not create new plugin on enter etc
+        onKeyDown={(e) => e.stopPropagation()}
+        className="h-32 w-full m-auto p-side resize-y font-mono shadow-menu border-none"
       >
         {state.code.value}
-      </Textarea>
-      <EditorInlineSettings>
```

Comment:
`EditorInlineSettings` is really just a div with a margin top, nothing more - good to know...

Checked it right now to be sure (https://github.com/edtr-io/edtr-io/blob/d1d3fd8e32c43ed9ec831b230b9147b7b85d54a4/packages/public/editor-ui/src/editor-inline-settings.tsx)

Reply:
jup, I was a bit underwhelmed as well ^^